README.md

```diff
@@ -55,41 +54,85 @@ Prior to running, ensure you have the following requirements installed:
 
-NOTE: This will also sync _*this*_ repo. For bootstrapping, we recommend running the script prior to cloning the repo and letting the script handle that.
+You will need your python3 path to make sure you environment is pointing to the right python version. You can find that out by running:
+```
+which python3
+```
 
-#### Vagrant
-There is also a Vagrantfile and provisioning shell script available. This will create an Ubuntu 14.04 virtual machine, provisioned with all the requirements to run the bootstrap script.
+To use virtualenv and virtualenv wrapper, in your terminal run:
+```
+mkvirtualenv open-fec-api --python </path-to-your-python3>
+```
 
-From scripts/bootstrap, simply:
+Subsequently, you will want to activate the environment every time you run the project with:
+```
+workon open-fec-api
+```
+### Install requirements
 
-    $ vagrant up
-    $ vagrant ssh
-    $ cp /vagrant/fec_bootstrap.sh fec_bootstrap.sh && ./fec_bootstrap.sh
+Use pip to install the requirements for the repo in your python3 environment:
+```
+pip install -r requirements.txt
+```
+#### Git hooks
+This repo includes optional post-merge and post-checkout hooks to ensure that
+dependencies are up to date. If enabled, these hooks will update Python
+dependencies on checking out or merging changes to `requirements.txt`. To
+enable the hooks, run:
+```
+invoke add_hooks
+```
+To disable, run:
+```
+invoke remove_hooks
+```
 
-#### Running the apps using tmuxinator
-Assuming you ran the bootstrap script, you can launch the API and the Web App with a single command:
+### Creating a new test database
+To run the API locally you will need a database, we provide a sql dump that has a small sample of data to get you running locally.
 
-    $ tmuxinator fec-local
+To create your own test database, make sure you have postgres and run:
+```
+createdb cfdm_test
+pg_restore --dbname cfdm_test data/subset.dump
+./manage.py update_all
+```
+(Don't worry if there are some user does not exist error messages)
 
-The site can be found at [http://localhost:3000](http://localhost:3000) (or [http://localhost:3001](http://localhost:3001) if using Vagrant). Remember the username and password you created when running the script.
+FEC and 18F members can set the sql connection to one of the RDS boxes with:
+```
+export SQLACONN=<psql:address-to-box>
+```
 
-#### Deployment
+### Run locally
+
+Run:
+```
+./manage.py runserver
+```
+The site can be found at [http://localhost:5000](http://localhost:5000)
+
+
+We are always trying to improve our documentation, if you have suggestions or run into problems feel free to [file an issue](https://github.com/18F/openFEC/issues).
+
+
+## Deployment
 
 ##### Likely only useful for 18F team members
 To deploy to Cloud Foundry, run `invoke deploy`. The `deploy` task will attempt to detect the appropriate
 Cloud Foundry space based the current branch; to override, pass the optional `--space` flag:
 
-    $ invoke deploy --space dev
+```
```

Comment:
Why change from indents to code fencing? AFAICT, they do the same thing. Just curious.